pypeit/par/util.py

```diff
@@ -26,6 +26,36 @@ def _eval_ignore():
     return [ 'open', 'file', 'dict', 'list', 'tuple' ]
 
 
+def eval_tuple(inp):
+    """
+    Evaluate the input to one or more tuples.
+
+    This allows conversion of one or more tuples provided to a configuration
+    parameters.
+
+    .. warning::
+        - Currently can only handle simple components that can also be evaluated
+          (e.g., integers and floats).
+
+    Args:
+        inp (:obj:`list`):
```

Comment:
Should this go in pypeit.utils instead?

Reply:
This function is currently only used by the next one (`recursive_dict_evaluate`).  I can move it, if you have a strong preference.